README.md

```diff
@@ -150,9 +153,11 @@ endif()
 
 ### Package Managers
 
+Regardless of the build system you use, when installing this library using the provided meson.build, a pkg-config file is installed. To use it, simply have your build system require the `nlohmann_json` pkg-config dependency.
```

Comment:
Please move this paragraph to the Meson paragraph.

Reply:
I would just like to reiterate that I believe this to be untrue, since pkg-config has nothing to do with whether your project uses meson (it only depends on whether nlohmann_json was installed using meson).

I believe moving the paragraph will be confusing to non-cmake users, but if you wish I can indeed move the paragraph...

Reply:
But if the installation of the pkg-config file depends on installing the library with meson, then why should this not be moved below?

Reply:
I'd have liked to separate the concerns of "how do I install nlohmann_json" vs. "how do I use the installed version".

Either way, I've updated the docs as you've requested.